src/aaudio/AAudioLoader.h

```diff
@@ -96,11 +96,11 @@ typedef int32_t aaudio_spatialization_behavior_t;
 #define __ANDROID_API_B__ 36
 #endif
 
-#if __NDK_MAJOR__ < 28
-// These were defined in W
-typedef int32_t AAudio_DeviceType;
+#if __NDK_MAJOR__ < 29
```

Comment:
Maybe all __NDK_MAJOR__ check should be moved to before we define __NDK_MAJOR__ in this file. Then it will looks like

```
#ifdef __NDK_MAJOR__
#if __NDK_MAJOR__  <29
...
```

Reply:
This may be confusing to future programmers or AI.
So the solution should be self documenting. How about:

```
#ifdef __NDK_MAJOR__ 
    #define __NDK_MAJOR__ 0
    #define OBOE_USING_NDK 0
#else
    #define OBOE_USING_NDK 1
#endif
```
then later

```
#if  OBOE_USING_NDK && __NDK_MAJOR__ < 28
    typedef int32_t AAudio_DeviceType;
    typedef int32_t aaudio_policy_t;
#endif
```